Casamari/MNC019.xml

```diff
@@ -0,0 +1,796 @@
+                                 <num value="2">፬</num>
+                                 <dim unit="leaf">10</dim>
+                                 <locus from="38r" to="47v"/>
+                              </item>
+                              <item xml:id="q6" n="5">  
+                                 <num value="2">፭</num>   
+                                 <dim unit="leaf">10</dim>
+                                 <locus from="48r" to="57v"/>
+                              </item>
+                              <item xml:id="q7" n="6">
+                                 <num value="2">፮</num>
+                                 <dim unit="leaf">9</dim>
+                                 <locus from="58r" to="66v"/>
+                              </item>
+                              <item xml:id="q8" n="7">
+                                 <num value="2">፯</num>
+                                 <dim unit="leaf">10</dim>
+                                 <locus from="67r" to="76v"/>
+                              </item>
+                              <item xml:id="q9" n="8">
+                                 <num value="2">፰</num>
+                                 <dim unit="leaf">10</dim>
+                                 <locus from="77r" to="86v"/>
+                              </item>
+                              <item xml:id="q10" n="9">
+                                 <num value="2">፱</num>
+                                 <dim unit="leaf">10</dim>
+                                 <locus from="87r" to="96v"/>
+                              </item>
+                              <item xml:id="q11" n="10">
+                                 <num value="2">፲</num>
+                                 <dim unit="leaf">10</dim>
+                                 <locus from="97r" to="106v"/>
+                              </item>
+                              <item xml:id="q12" n="11">
+                                 <num value="2">፲፩</num>
+                                 <dim unit="leaf">10</dim>
+                                 <locus from="107r" to="116v"/>
+                              </item>
+                              <item xml:id="q13" n="12">
+                                 <dim unit="leaf">10</dim>
+                                 <locus from="117r" to="126v"/>
+                              </item>
+                              <item xml:id="q14" n="13">
+                                 <dim unit="leaf">10</dim>
+                                 <locus from="127r" to="136v"/>
+                              </item>
+                              <item xml:id="q15" n="14">
+                                 <dim unit="leaf">8</dim>
+                                 <locus from="137r" to="144v"/>
+                              </item>
+                           </list>
+                        </collation>
+                        <condition key="deficient">
```

Comment:
Do we all assign "deficient" in the case of restoration?
Added to our agenda

Reply:
My statement was way too severe! In reality the Ms is in good condition.